frontend/src/components/gamesSection/gameCard/gameCard.tsx

```diff
@@ -67,10 +67,7 @@ export default class GameCard extends BaseCard<Game, GameCardProps, GameCardStat
     renderGameThumbnail(): JSX.Element {
         return (
             <div className="game-thumbnail-container">
-                {this.checkIfThumbnailPresent() ?
-                    <img className="game-thumbnail" alt={linkToString(this.props.object.gameLink)} src={this.props.object.thumbnail}/> :
-                    this.renderThumbnailPlaceholder()
-                }
+                {this.renderThumbnailPlaceholder()}
```

Comment:
but... this doesn't really fix the functionality. What if we have thumbnails later on in the database? This won't work with that, will it?

Reply:
True, I should probably be less lazy.